docs/synapse/userguides/ug017_storm_ref_stats.rst

```diff
@@ -1,16 +1,194 @@
-Storm Reference - Statistical Operators
-=======================================
+Storm Reference - Statistical Operator
+======================================
 
-Todo
+The statistical operator is used to generate data about data in Synapse.
+
+``stat()`` is defined in common.py_ as opposed to storm.py_.
+
+``stat()`` operates differently from other Storm operators:
+
+* ``stat()`` operates directly on the Synapse storage layer using the row-level APIs (as opposed to the node (form) APIs used by other Storm operators). This is an optimization that allows ``stat()`` to answer questions across large data sets ("all of the IP addresses in Synapse") that would otherwise be too "heavy" (non-performant) to lift.
+
+* Depending on the specific ``stat()`` handler used and the optimizations available in a particular Syanpse storage backing, the amount of time for a given ``stat()`` query to return may vary. For example, "count" operations will generally return much faster than "min" or "max" operations, even with the use of row-level APIs.
+
+* ``stat()`` is designed as a stand-alone operator; because it uses a different set of APIs, it cannot operate on the output of a previous Storm query and so cannot be "chained" as part of a larger query.
```

Comment:
This is really important to call out. What happens if you chain it together anyway?

Reply:
Based on quick and dirty testing it looks like Synapse simply ignores the stat() operator wherever it appears in the query (middle, end) but executes the other query elements as if the stat() operator were not present.